docs/user/alerting/geo-alert-types.asciidoc

```diff
@@ -0,0 +1,130 @@
+[role="xpack"]
+[[geo-alert-types]]
+== Geo alert types
+
+experimental[] Two additional stack alerts are available:
+<<alert-type-tracking-threshold>> and <<alert-type-tracking-containment>>. To enable,
+add the following configuration to your `kibana.yml`:
+
+```yml
+xpack.stack_alerts.enableGeoAlerting: true
+```
+
+As with other stack alerts, you need `all` access to the *Stack Alerts* feature
+to be able to create and edit either of the geo alerts.
+See <<kibana-feature-privileges, feature privileges>> for more information on configuring roles that provide access to this feature. 
+
+[float]
+=== Requirements to create a geo alert
+
+To create either a *Tracking threshold* or a *Tracking containment* alert, the
+following requirements must be present:
+
+- *Trackable index or index pattern data*: An index containing a `geo_point` field, `date` field,
```

Comment:
How about simplifying title to "Tracks index pattern"

Reply:
For consistency with the other alerting docs I should probably keep `index or index pattern`, but agreed on rest!